sdk-core/src/main/java/care/data4life/sdk/BaseClient.kt

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2020 D4L data4life gGmbH / All rights reserved.
+ *
+ * D4L owns all legal rights, title and interest in and to the Software Development Kit ("SDK"),
+ * including any intellectual property rights that subsist in the SDK.
+ *
+ * The SDK and its documentation may be accessed and used for viewing/review purposes only.
+ * Any usage of the SDK for other purposes, including usage for the development of
+ * applications/third-party applications shall require the conclusion of a license agreement
+ * between you and D4L.
+ *
+ * If you are interested in licensing the SDK for your own applications/third-party
+ * applications and/or if you’d like to contribute to the development of the SDK, please
+ * contact D4L by email to help@data4life.care.
+ */
+package care.data4life.sdk
+
+import care.data4life.sdk.SdkContract.LegacyDataClient
+import care.data4life.sdk.SdkContract.LegacyAuthClient
+import care.data4life.sdk.call.CallHandler
+import care.data4life.sdk.log.Log
+import care.data4life.sdk.log.Logger
+
+internal abstract class BaseClient(
+        protected var alias: String,
+        protected var userService: UserService,
+        protected var recordService: RecordService,
+        protected var handler: CallHandler,
+
+        private val legacyDataClient: care.data4life.sdk.LegacyDataClient = createLegacyDataClient(alias, userService, recordService, handler),
+        private val legacyAuthClient: LegacyAuthClient = createLegacyAuthClient(alias, userService, recordService, handler)
```

Comment:
Please chop both -> they exceed the current line limitation

Reply:
it's not a strict limit, if exceding the space is better readable this could be acceptable, but here it doesn't so chop chop